### DISCUSSION TITLE
Is it possible to have custom vehicle config when two mods use the same .xml name?

### DISCUSSION THREAD

**Discussion by coatsy35** [Courseplay/Courseplay_FS22#3348] [Q&A] [answered]
As per the subject - Is it possible to have custom vehicle config when two mods use the same .xml name? I have two different width mods which are based of the same original item and both have the same xml name. How do you handle this in custom config? is it possible?

**Comment by Tensuko** [comment, marked as answer]
- modName: Name of the .zip file (without '.zip')
    In case a Mod has the same .xml filename for the vehicle/implement, as a default giants vehicle/implement,
    add the name of the .zip file to prevent conflicts. For example: "FS22_exampleMod".
    Note: Only 2 of the same .xml files can be added.
